docs/examples/non_linear_pipeline_with_multiple_backends/README.md

```diff
@@ -0,0 +1,81 @@
+# Stable diffusion
+
+This example focuses on showcasing two of Triton Inference Server's features:
+* Using multiple frameworks in the same inference pipeline
+* Using the Python Backend API to build complex non linear pipelines.
+
+## How to run?
+
+Before starting, clone this repository and navigate to the root folder. It is recommended to use three different terminals for ease.
+
+### Step 1: Prepare the Server Environment
+* First, run the Triton Inference Server Container.
+```
+docker run --gpus=all -it --shm-size=256m --rm -p8000:8000 -p8001:8001 -p8002:8002 -v ${PWD}:/workspace/ -v ${PWD}/model_repository:/models nvcr.io/nvidia/tritonserver:22.08-py3 bash
+```
+* Next, we need to install all the dependencies required by the models running in the python backend.
```

Comment:
Try not to use "we", "I", and "you" statements. Just say, "Next, install the dependencies ..."